### PR TITLE
WD-8801 Up squared IoT grove server rebrand

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Go to https://ubuntu-com-13596.demos.haus/download/up-squared-iot-grove-server check against content on prod.
- Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/932?selectedIssue=WD-8801

## Design screenshot
managed

### CODE REVIEW

**Comment by juanruitina** [comment]
Please, turn "Install and develop snaps" into an H2 and drop "Next steps", it doesn't make sense when there's only one item under it. It should look like this:

Screenshot 2024-02-21 at 17 29 15

Other than that, LGTM:

**Comment by carkod** [comment]
> Please, turn "Install and develop snaps" into an H2 and drop "Next steps", it doesn't make sense when there's only one item under it. It should look like this:
> 
> Screenshot 2024-02-21 at 17 29 15
> Other than that, LGTM.

Done.

**Comment by juanruitina** [comment]
UX+1

File: templates/download/up-squared-iot-grove-server.html [hunk 1]
```diff
@@ -1,94 +1,77 @@
...
+        <h2 class="p-stepped-list__title">
           Download Ubuntu Server
-        </h3>
+        </h2>
         <div class="p-stepped-list__content">
           <p>Get the correct Ubuntu Server image for your board:</p>
-          <ul class="u-no-margin--left">
-            <li>Intel provides a modified Ubuntu Server 16.04.3 LTS image shipped with a custom 4.10 kernel and the Intel IoT Developer Kit.</li>
-            <li>Download and copy the <a href="https://downloads.up-community.org/download/up-squared-iot-grove-development-kit-ubuntu-16-04-server-image/">Ubuntu Server 16.04 LTS</a> image on a USB flash drive by following the live USB Ubuntu Desktop tutorial for <a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>, <a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>, or <a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a></li>
+          <ul class="p-list--divided u-no-margin--left">
```
Comment (mtruj013) [inline templates/download/up-squared-iot-grove-server.html:36]:
```suggestion
          <ul class="p-list--divided">
```
This isn't needed, Vanilla applies margin-left: 0 out of the box to divided lists
image

File: templates/download/up-squared-iot-grove-server.html [hunk 2]
```diff
@@ -1,94 +1,77 @@
...
-        </h3>
+        </h2>
         <div class="p-stepped-list__content">
           <p>Booting the board from the USB flash drive will start the Ubuntu installer.</p>
-          <ol class="u-no-margin--left">
-            <li>Insert the USB flash drive in the Up<sup>2</sup> board.</li>
-            <li>Power on the board.</li>
-            <li>The installer will start automatically. Follow the prompts to install Ubuntu Server on the on-board eMMC disk. You can refer to <a href="/tutorials/tutorial-install-ubuntu-server#4">this tutorial</a> for complete Ubuntu Server installation guidance.</li>
+          <hr class="p-rule" />
+          <ol class="p-list--divided u-no-margin--left">
```
Comment (mtruj013) [inline templates/download/up-squared-iot-grove-server.html:50]:
```suggestion
          <ol class="p-list--divided">
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just some minor comments

**Pull request merged**